### ISSUE TITLE
Seg fault with FDS case using CSVF

### ISSUE THREAD

**Issue by rmcdermo** [firemodels/fds#6362]
Glenn,
This case runs with latest FDS source on linux and the resulting output works with Smokeview.  If I run the case with official release of FDS 6.6.0 on my mac the case runs, but Smokeview seg faults.  Remove .txt from both file below to run the case.

[vortex_5.fds.txt](https://github.com/firemodels/fds/files/1967508/vortex_5.fds.txt)

[init_vortex_5_uvw.csv.txt](https://github.com/firemodels/fds/files/1967510/init_vortex_5_uvw.csv.txt)

**Comment by gforney** [comment]
I reproduced the problem with the smokeview and fds you were using.  Try smokeview here
https://drive.google.com/drive/folders/0B_wB1pJL2bFQc1F4cjJWY2duWTA?usp=sharing
This smokeview works for me on your case.

The smokeview you were using was built on a mac I don't normally use (was having trouble with my usual mac).  It has Intel 17 not 18.  The smokeview I just built was built with the "usual" mac using intel 18.  Also I made a few source changes since the smokview you were using.  

Let me know if this works.
